packages/now-routing-utils/test/superstatic.spec.js

```diff
@@ -534,12 +607,14 @@ test('convertHeaders', () => {
     ['hello/world/file.eot', 'another/font.ttf', 'dir/arial.font.css'],
     ['404.html'],
     ['/blog/first-post', '/blog/another/one'],
+    ['/like/params/first', '/like/params/second'],
   ];
 
   const mustNotMatch = [
     ['hello/file.jpg', 'hello/font-css', 'dir/arial.font-css'],
     ['403.html', '500.html'],
     ['/blogg', '/random'],
+    ['/non-match', '/like/params'],
```

Comment:
What about `/like/params/`. Should that match or not?

Reply:
It should not match since the param isn't optional, updated test to reflect this also